ui-backend/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/metacard/transformer/CsvTransformImpl.java

```diff
@@ -11,7 +11,7 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.catalog.ui.metacard.transform;
+package org.codice.ddf.catalog.ui.metacard.transformer;
```

Comment:
If you're going to change the package, might as well change the filename too imo. (`CsvTransformerImpl`)